.buildkite/pipeline.yml

```diff
@@ -0,0 +1,34 @@
+---
+steps:
+  - type: "waiter"
+
+  - name: ":docker:"
+    command: "bin/buildkite bin/ci build"
+    agents:
+      queue: native
+      docker: builder
+
+  - type: "waiter"
+
+  - name: ":pill:"
+    command: "bin/buildkite bin/ci run npm test"
+    agents:
+      queue: elastic
+
+  - type: "waiter"
+
+  - name: ":rocket: Publish"
+    type: "manual"
+    branches: "master"
+
+  - name: ":speech_balloon:"
+    command: "bin/buildkite bin/ci announce releasing-production"
+    branches: "master"
+    agents:
+      queue: elastic
+
+  - name: ":speech_balloon: Publish Treehorn"
+    command: "bin/buildkite bin/ci create-deploy-candidate production"
+    branches: "master"
+    agents:
+      queue: elastic
```

Comment:
Would be happy to.  I've still got a few bottles of nick kool-aid left over.  Let's crack one open and have a few glasses tomorrow perhaps.